pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -376,11 +383,22 @@ func (i *scaleUpdatedObjectInfo) UpdatedObject(ctx context.Context, oldObj runti
 		return nil, errors.NewNotFound(apps.Resource("deployments/scale"), i.name)
 	}
 
+	managedFieldsHandler := fieldmanager.NewScaleHandler(
+		deployment.ManagedFields,
+		schema.GroupVersion{Group: "apps", Version: "v1"},
```

Comment:
apps/v1 happens to be the only requestable version currently, this should really get the group/version from the request info in the context rather than assuming

that should be done as a follow-up, not in this PR

Reply:
Yes, I have forgotten to re-add it. I will create a separate PR for that.